src/components/presentations/Profilepage/ProfilePage.jsx

```diff
@@ -203,6 +194,47 @@ class Profilepage extends Component {
             <Userprofile isReviewer={profile && profile.is_reviewer} />
           ) : null}
         </div>
+        {userModalProfile ? (
```

Comment:
Rename this to `showModal` @sojida

Reply:
I don't think that would work... is `userModalProfile` an object?

Reply:
Yes, it is an object inside another object